app.rb

```diff
@@ -0,0 +1,134 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def app_options
+    puts '___________________________________________'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
```

Comment:
- [ ] I did not find a rental.rb file so I am leaving a comment here as you did not define rental so It's undefined please define it so that it would be easy to create rental
### Error
![sssssssssssssssssssssssss](https://user-images.githubusercontent.com/81727988/194687813-4a95bc2b-307e-40ea-a1c0-040b73af2e57.png)

Reply:
Can you explain to me why this error happened? Because I tested again everything, that error didn't show up...
![image](https://user-images.githubusercontent.com/47539402/194689676-a7f4f86d-5355-4817-81b7-08ff75699826.png)
![image](https://user-images.githubusercontent.com/47539402/194689698-fa8401c0-a186-42d5-8b0a-580eaf6002cf.png)
![image](https://user-images.githubusercontent.com/47539402/194689716-f3c48561-34a4-4168-80bc-fac762681c25.png)
![image](https://user-images.githubusercontent.com/47539402/194689746-77823fd4-8c7e-41c0-9474-c371addf433d.png)
![image](https://user-images.githubusercontent.com/47539402/194689792-825ff9dc-bab6-413c-a4bb-93678fd7d8fd.png)